File: aws-lambda-in-action/ch02/greetings-on-demand.py
import json

print('Loading function')

def lambda_handler(event, context):
    print('Received event: ' + json.dumps(event, indent=2))
    name = event['name'] if 'name' in event else 'World'
    print('name =', name);
    greetings = 'Hello ' + name + '!'
    print(greetings)
    return greetings

File: aws-lambda-in-action/ch02/greetings-on-demand.js
console.log('Loading function');

exports.handler = (event, context, callback) => {
    console.log('Received event: ', JSON.stringify(event, null, 2));
    console.log('name =', event.name);
    var name = ('name' in event) ? event['name'] : 'World';
    var greetings = 'Hello ' + name + '!';
    console.log(greetings);
    callback(null, greetings);
}